python/hsfs/client/exceptions.py

```diff
@@ -19,14 +19,18 @@ class RestAPIError(Exception):
     """REST Exception encapsulating the response object and url."""
 
     def __init__(self, url, response):
-        error_object = response.json()
+        try:
+            error_object = response.json()
```

Comment:
why was this needed?

Reply:
B/c it was failing if there is no body and not showing the error. Hsml does the same.